MediaBrowser.Controller/Plugins/IServerEntryPoint.cs

```diff
@@ -4,16 +4,22 @@
 namespace MediaBrowser.Controller.Plugins
 {
     /// <summary>
-    /// Interface IServerEntryPoint
+    /// Represents an entry point for a module in the application. This interface is scanned for automatically and
+    /// provides a hook to initialize the module at application start.
+    /// The entry point can additionally be flagged as a pre-startup task by applying the
```

Comment:
```suggestion
    /// The entry point can additionally be flagged as a pre-startup task by inheriting from the
```

Reply:
I changed this to "implementing". My understanding is that you can "apply" an attribute, "inherit" from a base class or "implement" an interface